cluster/deposit.go

```diff
@@ -45,3 +52,42 @@ func depositDataFromJSON(d depositDataJSON) DepositData {
 		Signature:             d.Signature,
 	}
 }
+
+// VerifyDepositAmounts verifies various conditions about partial deposits rules.
+func VerifyDepositAmounts(amounts []eth2p0.Gwei) error {
+	if len(amounts) == 0 {
+		return nil
+	}
+
+	const minSingleDepositAmountGwei = 1000000000
+	const maxTotalDepositAmountGwei = 32000000000
```

Comment:
These const values should probably be moved outside of this function to the top of the file